Library/Homebrew/cmd/deps.rb

```diff
@@ -22,9 +22,9 @@ def deps_args
         may be appended to the command. When given multiple formula arguments,
         show the intersection of dependencies for each formula.
       EOS
-      switch "-n",
+      switch "--topological", "-n",
              description: "Sort dependencies in topological order."
-      switch "--1",
+      switch "--skip-recursion", "--1",
```

Comment:
```suggestion
      switch "--direct", "--1",
```
Maybe?

Reply:
@carlocab mmm, I don't love it as we don't have any switches named like that. Any other suggestions?

Reply:
I think it kinda makes sense, since it's called as
```
brew deps --direct foo
```
Another alternative might be
```
brew deps --declared foo
```
Either would allow you to change the help text to something like
```
Show only the direct dependencies declared in the formula.
```
which is a lot clearer than the existing one, IMHO.

Reply:
Did both!